### ISSUE TITLE
Week 18 Review Ticket

### ISSUE THREAD

**Issue by Toby-Leeder** [Toby-Leeder/CSPFastpages#22]

**Comment by jm1021** [comment]
I don't think you were qualified to skip OOP work.  This project is not showing much in the way of design.  I have had people work on chess before and they did not function well in other projects as they became isolated programmers.

**Comment by chewyboba10** [comment]
- [x] Team. View and Frontend Hacks +++ Teachers looking to see beginnings of frontend coding in CPT project. 

I can see that they were starting to using css to help style parts of their website, particularly in the login/sign-up page that this review ticket links to. I would have like to see how they were developing the frontend of home page.

- [ ] Team. Model and OOP Hacks +++ Teachers looking to see beginnings of class definitions and object usage in CPT project.

Missing Model and OOP Hacks. The class code that you link to are in javascript and did not complete the hacks.

- [ ] Individual. Perform Create Performance Task Scoring blog +++ Teachers want to see understanding of criteria and evaluations of CPT. Also, we want to see how understanding scoring of these examples will help you exceed CPT requirements.

Poor reflection on Create Performance Task Scoring and you were missing the CPT Scoring for Submissions 3 & 4.
